engine/rpcserver_test.go

```diff
@@ -785,3 +786,135 @@ func TestGetHistoricTrades(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestGetAccountInfo(t *testing.T) {
+	bot := SetupTestHelpers(t)
+	s := RPCServer{Engine: bot}
+
+	accounts := refreshAccounts(t)
+
+	r, err := s.GetAccountInfo(context.Background(), &gctrpc.GetAccountInfoRequest{Exchange: fakePassExchange})
+	if err != nil {
+		t.Errorf("TestGetAccountInfo: Failed to get account info: %w", err)
```

Comment:
You can do t.Fatalf saves you doing return afterward.

Reply:
replaced